cleanlab_studio/studio/trustworthy_language_model.py

```diff
@@ -371,22 +376,29 @@ async def _prompt_async(
                 return None
             raise e
 
-        return {
-            "response": tlm_response["response"],
-            "trustworthiness_score": tlm_response["confidence_score"],
+        tlm_response = {
+            "response": response_json["response"],
+            "trustworthiness_score": response_json["confidence_score"],
         }
 
+        if self._return_logs:
+            tlm_response["logs"] = response_json["logs"]
+
+        return cast(TLMResponse, tlm_response)
+
     def get_trustworthiness_score(
         self,
         prompt: Union[str, Sequence[str]],
         response: Union[str, Sequence[str]],
-    ) -> Union[float, List[float]]:
+        logprobs: Optional[Union[float, Sequence[float]]] = None,
```

Comment:
```suggestion
        *,
        logprobs: Optional[Union[float, Sequence[float]]] = None,
```

Reply:
I think this should be prob, not probs (since it's just one number) 
or could consider argname `perplexity`

Reply:
make sure to apply these ^ changes everywhere

Reply:
Will change everything to `perplexity` (including the key name in the `log` dictionary).
Reason being that log probabilities are not on the 0-1 scale and are much less readable, I think it makes more sense to have something on the 0-1 scale (hence perplexity). Lmk if you have opinions otherwise.

Reply:
great but let's have this go into get_trustworthiness_score via kwargs only, so we dont need any documentation (they argname in kwargs can be perplexity)